Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM node:6.1.0-onbuild
+FROM node:alpine
```

Comment:
Would not be great there use a fix node alpine version instead get the latest one? eg: the latest TLS